modules/33acrossIdSystem.js

```diff
@@ -39,9 +39,11 @@ function calculateResponseObj(response) {
   };
 }
 
-function calculateQueryStringParams(pid, gdprConsentData) {
+function calculateQueryStringParams(pid, consentData) {
   const uspString = uspDataHandler.getConsentData();
-  const gdprApplies = Boolean(gdprConsentData?.gdprApplies);
+  const gdprApplies = Boolean(consentData?.gdprApplies);
+  const gppConsent = gppDataHandler.getConsentData();
```

Comment:
Out of curiosity would the API always guarantee a return value or is it possible for an error to be thrown?